### DISCUSSION TITLE
"failed to load temporary file for map"

### DISCUSSION THREAD

**Discussion by GeneralKarnage** [redeclipse/base#1181] [Help and Support]
I keep running into this message whenever I try to load and play any map, including the ones that come prepackaged with the game. The map will load, but then this message pops up and I am unable to spawn, being essentially trapped in spectator mode. I have tried uninstalling and reinstalling RE in the hopes that that might replace these "temporary files," but it did not work. Has anyone else ever encountered this problem, and how do I fix it?

I am running ver. 1.6.0 on a Dell gaming laptop running Windows 10.

**Comment by qreeves** [comment]
Might need to you provide your [log.txt](https://www.redeclipse.net/docs/FAQ#where-do-i-find-screenshots-logs-and-other-user-data). At this point, v1.6 is only receiving critical security fixes as we move toward the v2.0 sequel release on 19 December 2019. Jump on our Discord if you want a beta tester key for it on Steam.

**Comment by GeneralKarnage** [comment]
Thanks for the heads-up! Looking forward to the new release.

My log.txt is attached. [log removed after being downloaded by Quin]

**Comment by GeneralKarnage** [comment]
Problem still occurs, even in 2.0 beta. I can play on online servers, but when I try to play offline or play on a LAN server, the message pops up and I am unable to spawn. [log removed after being downloaded by Quin]

**Comment by SteveHeist** [comment]
Just an idea - do you have write access to the appropriate folder? (`C:/Users/[USER]/Documents/My Games/Red Eclipse`) You *should*, but double-checking never hurt anyone.

**Comment by no-lex** [comment]
You may want to scrub your personal information from your `log.txt`. In any case, this indeed looks like a permissions error to your home directory.

**Comment by qreeves** [comment]
Yep, this does seem to be a permissions error. Check `Documents\My Games\Red Eclipse` by right clicking that folder in Explorer and selecting `Properties`, go to the `Security` tab and ensure your user has write permission to the folder.

**Comment by GeneralKarnage** [comment]
I've tried that, but the problem still persists. I'll try to figure it out this weekend when I have time on my hands
